Assignment/solution_student/style.css

```diff
@@ -0,0 +1,37 @@
+header {
+	text-align: center;
+}
+
+h1 {
+	font-size: 40px; 
+	text-transform: uppercase; 
+}
+
+h3 {
+	text-transform: uppercase;
+}
+
+body {
```

Comment:
It's best practice to make the body declarations at the very top of the CSS document, since it's the highest-level _visible_ element on the page